Source/WebCore/platform/graphics/avfoundation/objc/SourceBufferPrivateAVFObjC.mm

```diff
@@ -758,6 +758,8 @@ static inline bool shouldAddContentKeyRecipients()
     if (!requiresFlush())
         return;
 
+    ALWAYS_LOG(LOGIDENTIFIER);
```

Comment:
Is it intentional to keep this logging?

Reply:
yes. the investigation of the bug was made difficult as we had logging from CoreMedia about when flushing is required; but no log whatsoever about when we are actually flushing.

This gave two very different branches of code to look into.

I assumed in this fix that flushing didn't actually occur and we were just stuck in the loop.

Had it been flushed, the bug wouldn't have been in WebKit.

Reply:
Great, that sounds reasonable. I just wanted to be sure! Sometimes it's easy to miss logging added for debugging when making the PR.